docs/Software Architecture Document/Component Interaction View/assets/app_launch.plantuml

```diff
@@ -0,0 +1,47 @@
+@startuml
```

Comment:
@AnastasiyaBritanova, this is the source code for `app_launch.png` image which included to `Component Interaction View`.